MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
+    
+    @State var selectedFarmer: FarmerPlace?
```

Comment:
Why did you make it optional? Is it possible to display that view without selected a farmer? 🤔

Reply:
I thought about it and it's more logical that it's not optional